cfme/base/login.py

```diff
@@ -15,6 +15,9 @@ class BaseLoggedInPage(View):
     )
     help = NavDropdown('.//li[./a[@id="dropdownMenu1"]]|.//li[./a[@id="help-menu"]]')
     settings = NavDropdown('.//li[./a[@id="dropdownMenu2"]]')
+    # 5.9 Locator for Settings item that replaces current group name when user has multiple groups
+    group_list_locator = (
```

Comment:
I really think this, and the code in the methods below using it, should be turning into a manageiq_widgetastic widget.  That would move the browser.element use out of this view model, and into the widget model where I believe it should be.

That would allow for defining properties or methods for the actions taken below, like:
* get current group when there are multiple
* determine if there are multiple via is_displayed instead of a NoSuchElementException on a browser.element call
* a widget.read() method that does the current_group_marker replacement
* definition of current_group_marker as a widget attribute that's usable outside of this View class property
* returning list of available groups in the widget

Reply:
This widget would implement `fill()`, and could be used to switch groups as well as reading.

Reply:
@LandoCalrizzian I see TODO added, acceptable to me!